upload/library/Sidane/Threadmarks/ControllerPublic/Post.php

```diff
@@ -33,12 +33,12 @@ public function actionThreadmark()
           $phrase = 'threadmark_deleted';
           XenForo_Model_Log::logModeratorAction('post', $post, 'delete_threadmark', array(), $thread);
         } else {
-          $threadmarksModel->updateThreadmark($existingThreadmark, $label);
+          $threadmarksModel->SetThreadMark($thread['thread_id'], $post['post_id'], $label);
```

Comment:
So this works even though the method is defined as `setThreadMark` but called here with a capital S?

Please rename it to `setThreadMark` anyway for consistency (and on line line 41 as well).